Library/Formula/rtags.rb

```diff
@@ -0,0 +1,74 @@
+
+class Rtags < Formula
+  homepage "https://github.com/Andersbakken/rtags"
+
+  stable do
+    url "https://github.com/Andersbakken/rtags/archive/v1.1.tar.gz"
+    sha1 "305f722b4fa1fc06954be31483d01c129083de1c"
+    patch :DATA
```

Comment:
Cool! Please add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!